test/onigumo_cli_test.exs

```diff
@@ -13,6 +13,12 @@ defmodule OnigumoCLITest do
     "-c"
   ]
 
+  @invalid_switch_combinations [
```

Comment:
These are not switch combinations. Switches and positional arguments are combined.

```suggestion
  @invalid_combinations [
```